### DISCUSSION TITLE
Allowing CRF > 30

### DISCUSSION THREAD

**Discussion by MaxiPoivre** [nagadomi/nunif#236] [Ideas]
Hi,
Conversion of 4K videos gives too big files, even at CRF=30.
Could you remove your CRF limit, in order to allow higher values ?
Thanks.

**Comment by nagadomi** [comment]
Did you try libx265 and Preset=medium? 

>Default h264: 933 MB
>h264 with preset "medium": 449 MB (artifacts reduced but still present)
>h265 with preset "medium": 275 MB (no artifacts when playback)
> https://github.com/nagadomi/nunif/issues/203#issuecomment-2293803303

**Reply by MaxiPoivre** [reply]
Usually I encode h265, medium.
Now I tried "fast" and I noticed a size increase by 20% ! I did not know that this preset (fast/medium) had an impact on the file size.
Usually I re-encode my 4k videos using CRF=35, with an acceptable quality. That's why maybe you could extend your limits ?

**Comment by nagadomi** [comment]
Unlocked CRF range 0-51 in GUI.

Personally, I think CRF35 will produce horrible artifacts, so if file size is a concern, it is better to downscale to FHD. https://github.com/nagadomi/nunif/discussions/227

**Reply by MaxiPoivre** [reply]
Thanks for the update. Indeed CR35 is bad for HD, but good for 4K.
4K can be more compressed than HD without any artefacts.
